react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const pokedex = [
+    { number: '001', name: 'Bulbasaur' },
+    { number: '004', name: 'Charmander' },
+    { number: '007', name: 'Squirtle' },
+    { number: '025', name: 'Pikachu' },
+    { number: '039', name: 'Jigglypuff' }
+  ];
+
+const pokemonList = pokedex.map((pokemon) => 
+    <li key={pokemon.name}>
```

Comment:
this should use the number rather than the name. We want that unique identifier!